easybuild/easyconfigs/b/BBMap/BBMap-39.01-GCC-12.3.0.eb

```diff
@@ -0,0 +1,33 @@
+easyblock = 'MakeCp'
+
+name = 'BBMap'
+version = '39.01'
+
+homepage = 'https://sourceforge.net/projects/bbmap/'
+description = "BBMap short read aligner, and other bioinformatic tools."
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = [SOURCEFORGE_SOURCE]
+sources = ['%(name)s_%(version)s.tar.gz']
+checksums = ['98608da50130c47f3abd095b889cc87f60beeb8b96169b664bc9d849abe093e6']
+
+dependencies = [
+    ('Java', '23.0.1', '', SYSTEM),
```

Comment:
we tend to stick to `('Java', '11', '', SYSTEM),` for the `12.3.0` generation, since this is compatible with most of the `Java`-based applications that easybuild supports for that toolchain generation